app/views/ShareListView/Header/Header.android.tsx

```diff
@@ -24,7 +23,13 @@ const styles = StyleSheet.create({
 	}
 });
 
-const Header = React.memo(({ searching, onChangeSearchText, theme }) => {
+interface IHeader {
+	searching: boolean;
+	onChangeSearchText(text: string): void;
```

Comment:
I always have my doubts on this case if we should just rename the function (so we can just extend `TextInputProps`) or use `onChangeSearchText: TextInputProps['onChangeText']` in order to keep the types as the component actually receives (of course, we know it's a function that takes `text: string` as param and returns `void`, but that doesn't mean it could change in the future). We should keep this as future proof as possible, so either you can rename `onChangeSearchText` to `onChangeText` and extend `TextInputProps` or make it `onChangeSearchText: TextInputProps['onChangeText']`

Reply:
I fixed this with https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3459/files#diff-57cd459b4f487e2435bae7298f83fba2a0ca18d7c60c1d965bbffd27cf9ac304R3